spec/features/admin_authentication_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.feature 'Admin Authentication', type: :feature do
+  scenario 'admin login successfully' do
+    Administrator.create(username: "someone@example.tld", password: "somepassword")
```

Comment:
In most specs, I'll use `create!` so the spec fails fast if it does not create the record as I'm assuming.